src/project.rs

```diff
@@ -263,7 +394,77 @@ impl FuzzProject {
         cmd.arg("-minimize_crash=1")
             .arg(format!("-runs={}", tmin.runs))
             .arg(&tmin.test_case);
-        exec_cmd(&mut cmd).with_context(|| format!("could not execute command: {:?}", cmd))?;
+
+        let old_artifacts = self.get_artifact_paths(&tmin.target)?;
```

Comment:
feels like the bulk of the work in fuzz and tmin can be merged into a common function? i'm okay either way

Reply:
sort of -- they're frustratingly similar, but just different enough that I wasn't convinced we could actually save any lines of code / make things more readable by factoring bits out